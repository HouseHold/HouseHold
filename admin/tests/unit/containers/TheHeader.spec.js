import Vue from 'vue'
import CoreuiVue from '@coreui/vue'
import TheHeader from '@/containers/TheHeader'
import { shallowMount } from '@vue/test-utils';

Vue.use(CoreuiVue)

describe('TheHeader.vue', () => {
  test('renders correctly', () => {
    const wrapper = shallowMount(TheHeader)
    expect(wrapper.element).toMatchSnapshot()
  })
})