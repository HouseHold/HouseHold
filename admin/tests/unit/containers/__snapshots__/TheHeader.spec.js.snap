// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheHeader.vue renders correctly 1`] = `
<cheader-stub
  colorscheme="light"
  fixed="true"
  light=""
  tag="header"
  withsubheader="true"
>
  <ctoggler-stub
    class="ml-3 d-lg-none"
    inheader="true"
    tag="button"
  />
   
  <ctoggler-stub
    class="ml-3 d-md-down-none"
    inheader="true"
    tag="button"
  />
   
  <cheaderbrand-stub
    alt="CoreUI Logo"
    class="mx-auto d-lg-none"
    height="46"
    src="img/brand/coreui-vue-logo.svg"
    width="190"
  />
   
  <cheadernav-stub
    class="d-md-down-none mr-auto"
  >
    <cheadernavitem-stub
      class="px-3"
    >
      <cheadernavlink-stub
        activeclass="active"
        event="click"
        exactactiveclass="active"
        routertag="a"
        target="_self"
        to="/dashboard"
      >
        
        Dashboard
      
      </cheadernavlink-stub>
    </cheadernavitem-stub>
     
    <cheadernavitem-stub
      class="px-3"
    >
      <cheadernavlink-stub
        activeclass="active"
        event="click"
        exact="true"
        exactactiveclass="active"
        routertag="a"
        target="_self"
        to="/users"
      >
        
        Users
      
      </cheadernavlink-stub>
    </cheadernavitem-stub>
     
    <cheadernavitem-stub
      class="px-3"
    >
      <cheadernavlink-stub
        activeclass="active"
        event="click"
        exactactiveclass="active"
        routertag="a"
        target="_self"
      >
        
        Settings
      
      </cheadernavlink-stub>
    </cheadernavitem-stub>
  </cheadernav-stub>
   
  <cheadernav-stub
    class="mr-4"
  >
    <cheadernavitem-stub
      class="d-md-down-none mx-2"
    >
      <cheadernavlink-stub
        activeclass="active"
        event="click"
        exactactiveclass="active"
        routertag="a"
        target="_self"
      >
        <cicon-stub
          name="cil-bell"
        />
      </cheadernavlink-stub>
    </cheadernavitem-stub>
     
    <cheadernavitem-stub
      class="d-md-down-none mx-2"
    >
      <cheadernavlink-stub
        activeclass="active"
        event="click"
        exactactiveclass="active"
        routertag="a"
        target="_self"
      >
        <cicon-stub
          name="cil-list"
        />
      </cheadernavlink-stub>
    </cheadernavitem-stub>
     
    <cheadernavitem-stub
      class="d-md-down-none mx-2"
    >
      <cheadernavlink-stub
        activeclass="active"
        event="click"
        exactactiveclass="active"
        routertag="a"
        target="_self"
      >
        <cicon-stub
          name="cil-envelope-open"
        />
      </cheadernavlink-stub>
    </cheadernavitem-stub>
     
    <theheaderdropdownaccnt-stub />
  </cheadernav-stub>
   
  <csubheader-stub
    class="px-3"
    tag="div"
  >
    <cbreadcrumbrouter-stub
      class="border-0"
    />
  </csubheader-stub>
</cheader-stub>
`;
